dist/COVID-VACCINE-TRIAL-schema.json

```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Covid Vaccine Trial",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "email": {
+      "type": "string",
+      "minLength": 6,
+      "maxLength": 80,
+      "format": "email"
+    },
+    "fullName": {
```

Comment:
https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L5 ?

Reply:
@mchelen-gov fullName is from common defs.